Pod/Classes/CZPickerView.m

```diff
@@ -308,7 +308,11 @@ - (UITableViewCell *)tableView:(UITableView *)tableView cellForRowAtIndexPath:(N
             cell.accessoryType = UITableViewCellAccessoryCheckmark;
         }
     }
-    if ([self.dataSource respondsToSelector:@selector(czpickerView:attributedTitleForRow:)]) {
+    if([self.dataSource respondsToSelector:@selector(czpickerView:titleForRow:)] && [self.dataSource respondsToSelector:@selector(czpickerView:titleForRow:)]){
```

Comment:
Sorry... it's a type error... I'll correct it and re-push to you, sorry for it.
image + attributedTitle Yes, I'll add it and re-push to you ;)